src/EventStore.Core/ClusterVNode.cs

```diff
@@ -1867,6 +1871,25 @@ public class ClusterVNode<TStreamId> :
 			}
 		}
 
+		private static bool TryRunningPublicTcpApi(out int port, out int advertisedPort) {
```

Comment:
in here we should probably log if we are enabling this unsupported feature

Reply:
Done